umbral/keys.py

```diff
@@ -152,6 +192,28 @@ def to_bytes(self):
     def get_pubkey(self):
         raise NotImplementedError
 
+    def to_cryptography_pubkey(self):
+        """
+        Returns a cryptography.io EllipticCurvePublicKey from the Umbral key.
+        """
+        backend = default_backend()
+
+        backend.openssl_assert(self.point_key.group != backend._ffi.NULL)
+        backend.openssl_assert(self.point_key.ec_point != backend._ffi.NULL)
+
+        ec_key = backend._lib.EC_KEY_new()
+        backend.openssl_assert(ec_key != backend._ffi.NULL)
+        ec_key = backend._ffi.gc(ec_key, backend._lib.EC_KEY_free)
```

Comment:
Is it not possible / practical to do `_lib.EVP_PKEY_free` here instead?

Reply:
Why would I do that? It's an EC_KEY?

Reply:
Yyyyeahhh, I think I was kinda wilin' there.  It was late.  :-)

I was imagining a flow like:

```
pkey = _lib.EVP_PKEY_new()
self._pkey = _ffi.gc(pkey, _lib.EVP_PKEY_free)
backend._lib.EC_KEY_new()
...
backend._lib.EVP_PKEY_assign_EC_KEY(...)
```

But yeah, I see now that that does not make sense.